### PR TITLE
ZH-trans: add overview.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/kubectl/overview.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ where `command`, `TYPE`, `NAME`, and `flags` are:
 <!--
 * `command`: Specifies the operation that you want to perform on one or more resources, for example `create`, `get`, `describe`, `delete`.
 
-* `TYPE`: Specifies the [resource type](#resource-types). Resource types are case-insensitive and you can specify the singular, plural, or abbreviated forms. For example, the following commands produce the same output:
+* `TYPE`: Specifies the [资源类型](#resource-types). Resource types are case-insensitive and you can specify the singular, plural, or abbreviated forms. For example, the following commands produce the same output:
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]: why this change?

Comment (xichengliudui) [reply content/zh/docs/reference/kubectl/overview.md]: Annotation error.

File: content/zh/docs/reference/kubectl/overview.md [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+$ kubectl exec <pod-name> -c <container-name> date
+
+// 获取一个互动 TTY 和运行 /bin/bash <pod-name >。默认情况下，输出来自第一个容器。
+$ kubectl exec -ti <pod-name> /bin/bash
+```
+
+<!--
+`kubectl logs` - Print the logs for a container in a pod.
+-->
+`kubectl logs` - 在容器中打印容器的日志。
```
Comment (SataQiu) [inline content/zh/docs/reference/kubectl/overview.md]: 不是在容器中打印
```suggestion
`kubectl logs` - 打印 Pod 中容器的日志。
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+
+// 我们可以"卸载"一个插件，只需从我们的路径中删除它
+$ sudo rm /usr/local/bin/kubectl-hello
+```
+
+<!--
+In order to view all of the plugins that are available to `kubectl`, we can use
+the `kubectl plugin list` subcommand:
+-->
+为了查看可用的所有插件 `kubectl`，我们可以使用 `kubectl plugin list`子命令：
```
Comment (SataQiu) [inline content/zh/docs/reference/kubectl/overview.md]:
```suggestion
为了查看可用的所有 `kubectl` 插件，我们可以使用 `kubectl plugin list` 子命令：
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 4]
```diff
@@ -0,0 +1,804 @@
...
+```
+
+<!--
+Running the above plugin gives us an output containing the user for the currently selected
+context in our KUBECONFIG file:
+-->
+运行上面的插件可以得到一个输出，其中包含当前在 KUBECONFIG 文件中选择的上下文的用户:
+
+```shell
+// make the file executable
```
Comment (SataQiu) [inline content/zh/docs/reference/kubectl/overview.md]: 没翻译

File: content/zh/docs/reference/kubectl/overview.md [hunk 5]
```diff
@@ -0,0 +1,804 @@
...
+$ sudo chmod +x ./kubectl-whoami
+
+// and move it into our PATH
```
Comment (SataQiu) [inline content/zh/docs/reference/kubectl/overview.md]: 没翻译

**Comment by tengqm** [comment]
ping @xichengliudui This one has been here for a long time.

**Comment by xichengliudui** [comment]
I forgot it, wait a minute, I will update this pr at noon
